examples/example-node/server.ts

```diff
@@ -23,11 +25,29 @@ const posthog = new PostHog(PH_API_KEY, {
 
 posthog.debug()
 
+Sentry.init({
+  dsn: 'https://examplePublicKey@o0.ingest.sentry.io/0',
+
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0,
```

Comment:
I think you meant https://docs.sentry.io/platforms/node/configuration/options/#sample-rate instead?
`tracesSampleRate` is for transactions, `sampleRate` is for errors which is this integration for.

Reply:
I just copy pasted their default :D